README.md

```diff
@@ -59,21 +60,49 @@ The container system is [Docker](https://www.docker.com/).
 The build system is [Gradle](https://gradle.org/) and it supports all of the above.
 
 ## IntelliJ configuration
+
 The [Lombok plugin](https://plugins.jetbrains.com/plugin/6317-lombok/) is required for IntelliJ, else the code generated
 by the Lombok annotations will not be visible (and the project will be littered with red squiggle line errors).
 
 ## Building
+
 To build the entire application, including running unit and functional tests:
 
-`./gradlew build`
+`./gradlew startServer && ./gradlew build`
 
 (Note that functional tests share the same port number for embedded database as for the containerised database, if
-tests fail try running `./gradlew composeDown` first).
+tests fail try running `./gradlew composeDown` first. Free-up port for the keycloak test server as well).
+
+Create a `.env` file in the project root directory and do the env variables setup for keycloak:
+
+```
+KEYCLOAK_SERVER_PORT=8180
+KEYCLOAK_USER=admin@everest.engineering
+KEYCLOAK_PASSWORD=******
+```
+
+(Note: the above env variable values should match with the keycloak server details specified in the `application.properties` file and the `build.gradle` file present in the launcher module.
+Keycloak test server also uses the same credentials for authentication.)
```

Comment:
We have introduced a default `.env` file in the project root directory to increase the ease of use.

Packaging with a script or a gradle task may not be feasible here as it involves changing values in multiple(`application.properties` and `gradle.build`) places.
Fixing https://github.com/everest-engineering/lhotse/issues/51 issue will avoid test server details matching case.
Created a new issue https://github.com/everest-engineering/lhotse/issues/54 to avoid changing values in multiple places.